prow/plugins/config.go

```diff
@@ -169,6 +169,16 @@ type Blunderbuss struct {
 	IgnoreAuthors []string `json:"ignore_authors,omitempty"`
 }
 
+// OrgBlunderbuss allows sharding for org and repo-specific Blunderbuss settings,
+// providing Blunderbuss settings under either org or org/repo keys.
+type OrgBlunderbuss map[string]BlunderbussConfig
+
+// Blunderbuss defines overall configuration for the blunderbuss plugin.
+type Blunderbuss struct {
```

Comment:
Why just not include the new `OrgBlunderbuss` in the "old" `BlunderbussConfig`?

Reply:
I like having a single struct referenced everywhere so that if an update was required, it only needs to be updated in one place and you get it in both places for free, but I can definitely change it if that's preferred since it's a minor thing.

Reply:
I would vote for changing it and including the new type but I do not have a super strong opinion.

Reply:
Oh, I see what you mean--having `OrgBlunderbuss` separate doesn't provide any value. So something like this?

```golang
// Blunderbuss defines overall configuration for the blunderbuss plugin.
type Blunderbuss struct {
	BlunderbussConfig
	// OrgBlunderbuss allows sharding for org and repo-specific Blunderbuss settings,
	// providing Blunderbuss settings under either org or org/repo keys.
	OrgBlunderbuss map[string]BlunderbussConfig
}
```

Reply:
Ah, furthermore I was hoping to be able to do something like:
```yaml
blunderbuss:
  <config>
  "org/repo":
    <config>
```
But that's actually not possible because having an anonymous `map[string]` inside a struct would invite the possibility to generate conflicting fields, so this is indeed the path forward.